debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnectorConfig.java

```diff
@@ -123,8 +123,14 @@ public static SnapshotMode parse(String value, String defaultValue) {
         SNAPSHOT("snapshot"),
 
         /**
-         * This mode will avoid using ANY table locks during the snapshot process.  This mode can only be used with SnapShotMode
-         * set to schema_only or schema_only_recovery.
+         * This mode will avoid taking any table locks during the snapshot process.  As long as DDL statements are not
+         * executed during snapshot process, data consistency is preserved.
+         */
+        CONSISTENT("consistent"),
```

Comment:
@grzegorz8 How about naming this after the locking mode it uses, i.e. `REPEATABLE_READ`? Makes semantics much clearer, plus, repeatable read isolation still allows for phantom reads, i.e. someone might argue that this doesn't fully guarantee consistency.

Reply:
And should this be the default? It seems much more sensible than `NONE` (and it is more in line with the previous default behavior, where `NONE` was using repeatable read semantics). In fact, is there any reason for keeping `NONE` at this point?

Reply:
I agree with both.